content/dk/mada/style/mada-style-gradle/mada-style-gradle-0.9.9.buildinfo

```diff
@@ -25,7 +25,7 @@ outputs.2.1.checksums.sha512=c40147bc923c2fb920f48a20904208f1a5a66b933bcafc4b1da
 
 outputs.2.2.filename=mada-style-gradle-0.9.9.module
 outputs.2.2.length=4103
-outputs.2.2.checksums.sha512=5537207a6e2f4dc1fa8bff009ded4730b87b19145c207867247a7451f3777749774c3d304ecadd3b0427125bc608ca1b7c9c7cb616bf21788ca7ff36273e6df8
```

Comment:
you'll need to rm the `.diffoscope` file too, and comment it in the `.buildspec`

Reply:
righto